kvm-install-vm

```diff
@@ -691,6 +702,10 @@ _EOF_
     outputn "Cleaning up cloud-init files"
     rm -f $USER_DATA $META_DATA $CI_ISO && ok
 
+    if [ -n "${NETWORK_NAME}" ]; then
+        BRIDGE=$(virsh net-dumpxml "${NETWORK_NAME}" | awk -F\' '/bridge/ {print $2}')
```

Comment:
I'm a bit nervous about splitting on the first single quote. What if `name=` is not first in the bridge tag?

What about something like: `virsh net-info default | awk -F':' '/Bridge/ {print $2}' | tr -d ' '` ?

Thoughts?